orchagent/portsorch.cpp

```diff
@@ -3431,6 +3462,14 @@ bool PortsOrch::removeBridgePort(Port &port)
         return false;
     }
 
+    //Flush the FDB entires corresponding to the port
```

Comment:
How about [removeDefaultBridgePorts](https://github.com/Azure/sonic-swss/blob/c165697df2124d971a1c80254e1100c760e99fde/orchagent/portsorch.cpp#L447)

Reply:
addressed in the next commit.